box.json.dist

```diff
@@ -0,0 +1,13 @@
+{
+    "output": "phpbrew",
+    "compression": "GZ",
+    "compactors": [
+        "KevinGH\\Box\\Compactor\\Json",
+        "KevinGH\\Box\\Compactor\\Php"
+    ],
+    "check-requirements": false,
```

Comment:
I guess the requirement checker is causing problems here?

Reply:
No, I just excluded it to make the result closer to what it used to be previously. When we bump the PHP requirements, I'll consider enabling it back.

Reply:
>No, I just excluded it to make the result closer to what it used to be previously. 

Makes total sense

>When we bump the PHP requirements, I'll consider enabling it back.

IMO it's more about when you want to give it a try. Even if you don't bump the PHP requirements, it also checks extensions provided they are properly listed in the `composer.json`

Reply:
This is very true. It was just out of scope. I'm totally open to enabling it and making sure all required extensions are listed in composer.json.